docs/content/user-manual/elasticjob-lite/configuration/built-in-strategy/error-handler.cn.md

```diff
@@ -47,19 +47,18 @@ Maven 坐标：
 
 可配置属性：
 
-| 属性名          | 说明        | 默认值                    |
-| -------------- |:----------- |:------------------------ |
-| email.host     | TODO        | 无                       |
-| email.port     | TODO        | 无                       |
-| email.username | TODO        | 无                       |
-| email.password | TODO        | 无                       |
-| email.useSsl   | TODO        | true                     |
-| email.subject  | TODO        | ElasticJob error message |
-| email.from     | TODO        | 无                       |
-| email.to       | TODO        | 无                       |
-| email.cc       | TODO        | 无                       |
-| email.bcc      | TODO        | 无                       |
-| email.debug    | TODO        | false                    |
+| 属性名    | 说明                 | 默认值                   |
+| -------- |:-------------------- |:------------------------ |
+| host     | 邮件服务主机地址      | 无                       |
+| port     | 邮件服务主机端口      | 无                       |
+| username | 用户名               | 无                       |
+| password | 密码                 | 无                       |
+| useSsl   | 是否启用ssl加密传输   | 无                       |
```

Comment:
There is default value here

Reply:
I will correct this  and other same case.